server/backend/backend.go

```diff
@@ -272,7 +273,14 @@ func (b *Backend) GetAddressByHash(ctx context.Context, hash string) (*models.Ad
 	if err != nil {
 		return nil, err
 	}
-	address.NumberOfTransactions = int(nonce)
+	transactionCounter := int(nonce)
```

Comment:
Is nonce ever the right value?  Incoming transactions won't be counted if we use the nonce right?

Reply:
Updated, now using for everything